ssl/t1_trce.c

```diff
@@ -95,36 +95,35 @@ static ssl_trace_tbl ssl_handshake_tbl[] = {
 
 /* Cipher suites */
```

Comment:
Did you want a comment per cipher to the rfc or should one link to here be fine?
https://www.ietf.org/assignments/tls-parameters/tls-parameters.txt